plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/persistence/LibraryClasspathContainerSerializer.java

```diff
@@ -133,4 +143,15 @@ public IPath getContainerStateFile(IJavaProject javaProject, IPath containerPath
       return containerFile.getLocation();
     }
   }
+
+  private static class M2LocalRepositoryLocationProvider implements ArtifactBaseLocationProvider {
+
+    /* (non-Javadoc)
```

Comment:
why non-Javadoc? MIght as well make it Javadoc is only to avoid the extra comment :-)